### PR TITLE
Added Post Incident Practices module

### PR DESCRIPTION
**Pull request by tom-halpin**
In response to the reaction from the community to the **Online DevOps Dojo** happy to be able to announce the open sourcing of a module on Post Incident Practices.

### CODE REVIEW

**Comment by tom-halpin** [comment]
@cpswan @ojacques @pberthonneau open sourcing Post Incident Practices module by way of a small thank you to the community given the very positive reaction to the Dojo since we launched it.

File: CHANGELOG.md [hunk 1]
```diff
@@ -5,6 +5,12 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [1.1.2 - 2020-12-15]
```
Comment (adamdmharvey) [inline CHANGELOG.md:8]: For SemVer, any changes which aren't bug fixes maybe should bump the minor, so 1.2.0?

**Review by adamdmharvey — APPROVED** [review]
❤️

File: CHANGELOG.md (hunk 1 above)
Comment (tom-halpin) [reply CHANGELOG.md]: @adamdmharvey done, thanks

**Pull request merged**